### DISCUSSION TITLE
Synonym Postfix Snippet

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#903] [Share]
https://user-images.githubusercontent.com/36073/193135022-15284563-f259-4aba-a8d7-f9f0181db103.mp4

[Open synonym-snippet in Script Kit](https://scriptkit.com/api/new?name=synonym-snippet&url=https://gist.githubusercontent.com/johnlindquist/68e848629c110f3f685d5434033afade/raw/3cc3a815ea94e63aca139b100fdb5118065bb7d5/synonym-snippet.ts")

```js
// Name: Synonym Snippet
// Snippet: *,,

import "@johnlindquist/kit"

let word = await arg("The value of *")

let response = await get(`https://api.datamuse.com/words?ml=${word}`)

let synonyms = response.data.map(i => i.word)

let synonym = await arg("Select a replacement", synonyms)

await setSelectedText(synonym)


```

**Comment by benjaminmodayil** [comment]
How does that snippet fire? I've never saw that syntax before.

**Reply by johnlindquist** [reply]
Type ",," to trigger the snippet. "*" Tells the script to grab the text you typed before it and pass it to "arg" so you can manipulate it
